src/components/Notifications/NotificationTypes/BidAcceptedNotification.tsx

```diff
@@ -60,13 +61,23 @@ const BidAcceptedNotification = ({
       }}
       timestamp={notification.timestamp}
       isNew={!notification.read}
+      locale={locale}
     >
       <p className="dcl notification-item__content-title">
         {i18N[locale].title}
       </p>
       <p className="dcl notification-item__content-description">
         {i18N[locale].description(
-          formatMana(notification.metadata.price),
+          <Mana
+            inline
+            network={
+              notification.metadata.network === 'polygon'
```

Comment:
What do you think about extraction to a enum?

Reply:
we already have an enum for Network in @dcl/schemas but it doesn't work for this. I prefer to keep it as is because it only belongs to Notifications domain.